src/main/webapp/resources/js/pages/cart/index.js

```diff
@@ -1,16 +1,17 @@
+import { configureStore } from "@reduxjs/toolkit";
 import React from "react";
 import { render } from "react-dom";
 import { Provider } from "react-redux";
-import { getStore } from "../../redux/getStore";
-import { actions } from "../../redux/reducers/app";
-import {
-  empty,
-  loadFullCart,
-  removeProjectFromCart,
-  removeSampleFromCart,
-} from "../../redux/sagas/cart";
-import { Cart } from "./components/Cart";
+import { cartApi } from "../../apis/cart/cart";
+import { projectsApi } from "../../apis/projects/projects";
 import { setBaseUrl } from "../../utilities/url-utilities";
+import { Cart } from "./components/Cart";
+import shareReducer from "./components/share/shareSlice";
```

Comment:
This can be removed as its no longer used

Reply:
REmoved